mslice/scripting/__init__.py

```diff
@@ -0,0 +1,13 @@
+from mantid.simpleapi import GeneratePythonScript
+from mslice.models.workspacemanager.workspace_provider import get_workspace_handle
+from mslice.util.qt import QtWidgets
+
+
+def generate_script(plot_window, ws_name, filename):
```

Comment:
Maybe make `filename=None` a default to make clear it's optional?

Reply:
Also the more standard matplotlib terminology would be `figure` so maybe use that as an argument name?